salt/vault/secret_backends.sls

```diff
@@ -15,6 +17,28 @@ enable_mitx_aws_secret_backend:
     - lease_max: {{ SIX_MONTHS }}
     - lease_default: {{ SIX_MONTHS }}
 
+enable_pki_intermediate_backend:
+  vault.secret_backend_enabled:
+    - backend_type: pki
+    - mount_point: pki_int
```

Comment:
We are using `-` separators for our other backends so we should stick with that convention. Also, a more descriptive mount point would be helpful (e.g. `pki-intermediate-ca`)

Reply:
Might also be more descriptive (though not _technically_ accurate) to refer to this as the 'root' CA